### DISCUSSION TITLE
feed showing twice some time

### DISCUSSION THREAD

**Discussion by dhrn** [community/community#33737] [Other Feature Feedback, Questions, & Ideas] [answered]
<img width="795" alt="image" src="https://user-images.githubusercontent.com/14145706/191585230-5cb9c161-c3fd-4902-a1dc-e2ba36feb8e3.png">

**Comment by jfullstackdev** [comment, marked as answer]
I believe they are still refining this beta, like in the past, already starred repos are still appearing on Recommendations

**Comment by sholajapheth** [comment]
Have u made this complain... try writing to github.

**Comment by kaushikravuri** [comment]
May be a glitch, try to write to the github and hope it will be resolve and for me it's never happen

**Comment by dhrn** [comment]
🤷‍♂️  may be a glitch
